src/pages/Dashboard/MintRepayVai/MintVai/index.tsx

```diff
@@ -52,7 +52,7 @@ export const MintVaiUi: React.FC<IMintVaiUiProps> = ({
 
   const vaiToken = getToken(VAI_ID);
   const limitTokens = useMemo(
-    () => (limitWei ? convertWeiToCoins({ value: limitWei, tokenId: VAI_ID }).toString() : '0'),
+    () => (limitWei ? convertWeiToCoins({ valueWei: limitWei, tokenId: VAI_ID }).toString() : '0'),
     [limitWei?.toString()],
```

Comment:
Following my previous comment:
```suggestion
    () => (limitWei ? convertWeiToCoins({ valueWei: limitWei, tokenId: VAI_ID }).toFixed() : '0'),
    [limitWei?.toFixed()],
```

Reply:
solved with rebase